### ISSUE TITLE
Nested eager load

### ISSUE THREAD

**Issue by Venommmm** [src-d/go-kallax#197]
Is it possible, for example, to get all the chain: Restaurant - > Reviews - > User? I cant find a way to preload user for each review in this case.

**Comment by erizocosmico** [comment]
Nope, right now there is no way to do nested eager loading. Your case should be doable with a `through`, when support for that is added. It's being worked on, but it might still be a while until it's released.

**Comment by patrickdevivo** [comment]
curious about this issue as I'm running into a similar need. any updates here from the kallax maintainers?
